jvm/src/main/scala/fm/common/ClassUtil.scala

```diff
@@ -212,16 +212,17 @@ object ClassUtil extends Logging {
     val urls: Set[URL] = classLoader.getResources(packageDirPath).asScala.toSet
     
     urls.flatMap { url: URL =>
+      val filePath: String = URLDecoder.decode(url.getFile(), "UTF-8")
+
       url.getProtocol() match {
-        case "jar" => 
-          val fullJarUrl: String = URLDecoder.decode(url.getFile(), "UTF-8")
-          val jarFile: String = fullJarUrl.substring("file:".length, fullJarUrl.indexOf("!"))
-          val jarPrefix: String = fullJarUrl.substring(fullJarUrl.indexOf("!") + 1)
-          require(jarPrefix == File.separator+packageDirPath, s"Expected jarPrefix ($jarPrefix) to equal package prefix ($packageDirPath)")
-          scanJar(packageDirPath+File.separator, new File(jarFile))
+        case "jar" =>
+          val jarFile: String = filePath.substring("file:".length, filePath.indexOf("!"))
+          val jarPrefix: String = filePath.substring(filePath.indexOf("!") + 1)
+          require(jarPrefix == s"/$packageDirPath", s"Expected jarPrefix ($jarPrefix) to equal package prefix (/$packageDirPath)")
```

Comment:
The `jar` test remains the same, I think just the file-test is different.  I'll see about adding an explicit test for this.  I think I ran into this because on win32 I got back a  url.getFile() like: `c:/Users/Eric%20Peters/IdeaProject/fm-common/...` and it didn't work unless I url decoded the path before creating a `new File(path)` out of it.